src/EditorFeatures/CSharpTest/Completion/CompletionProviders/ExtensionMethodImportCompletionProviderTests.cs

```diff
@@ -513,6 +513,7 @@ public void M(MyType x)
         [Theory, Trait(Traits.Feature, Traits.Features.Completion)]
         public async Task ValueTupleType(string tupleType, ReferenceType refType)
         {
+            // TODO2
```

Comment:
?

Reply:
I'm still investigating this test failure:
```
    (8,55): error CS8356: Predefined type 'System.ValueTuple`2' is declared in multiple referenced assemblies: 'System.ValueTuple, Version=4.0.1.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' and 'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'
    (8,49): error CS8356: Predefined type 'System.ValueTuple`2' is declared in multiple referenced assemblies: 'System.ValueTuple, Version=4.0.1.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' and 'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'
```